packages/tables/src/Columns/Concerns/CanUpdateState.php

```diff
@@ -30,6 +32,8 @@ public function updateState(mixed $state): mixed
             ]);
         }
 
+        $this->callBefore();
```

Comment:
It's probably worth passing in `$this->evaluate(..., ['state' => $state])`

Reply:
done ✅